src/Microsoft.DotNet.RemoteExecutor/src/Microsoft.DotNet.RemoteExecutor.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCurrent);$(NetFrameworkMinimum)</TargetFrameworks>
+    <TargetFrameworks>$(NetMinimum);$(NetCurrent);$(NetFrameworkMinimum)</TargetFrameworks>
```

nit: the TFM order matters as VS picks the first one.

```suggestion
    <TargetFrameworks>$(NetCurrent);$(NetMinimum);$(NetFrameworkMinimum)</TargetFrameworks>
```